app-core/src/main/java/com/mercury/platform/core/utils/interceptor/TradeOutMessagesInterceptor.java

```diff
@@ -49,18 +49,18 @@ protected void process(String message) {
     protected MessageMatcher match() {
         return message ->
                 this.clients.stream()
-                        .filter(matcher -> matcher.isSuitableFor(message))
-                        .findAny().orElse(null) != null;
+                            .filter(matcher -> matcher.isSuitableFor(message))
+                            .findAny().orElse(null) != null;
     }
 
     private abstract class LocalizationMatcher {
         public boolean isSuitableFor(String message) {
             return message.contains("Hi, I would like") ||
-                    message.contains("Hi, I'd like") || message.contains("I'd like") ||
-                    message.contains("구매하고 싶습니다" /* "I would like to buy"*/) ||
-                    message.contains("Здравствуйте, хочу купить у вас") /* "Hello, I would like to buy" */ ||
-                    (message.contains("wtb") && message.contains("(stash") ||
-                    message.contains("안녕하세요, 강탈" /* hello, ???*/)); //pathofexile.com/trade/ // Is this "wtb" + korean correct? Also in TradeInc[...].java
+                   message.contains("Hi, I'd like") || message.contains("I'd like") ||
+                   message.contains("구매하고 싶습니다" /* "I would like to buy"*/) ||
+                   message.contains("Здравствуйте, хочу купить у вас") /* "Hello, I would like to buy" */ ||
+                   (message.contains("wtb") && message.contains("(stash")) ||
+                   message.contains("안녕하세요, 강탈" /* hello, ???*/); //pathofexile.com/trade/ // Is this "wtb" + korean correct? Also in TradeInc[...].java
```

Comment:
Delete comment `// Is this "wtb" + korean correct? Also in TradeInc[...].java`, the parenthesis was fixed.